gen/definitions/pnp_import_devices.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: PnP Import Devices
 rest_endpoint: /dna/intent/api/v1/onboarding/pnp-device/import
-res_description: 'This resource adds devices to PNP in bulk  file on the chosen network device (upgrade the software on the device).
+res_description: 'This resource adds devices to PNP in bulk.
   Every time this resource is created or re-created, the Catalyst Center considers adding new devices to PNP.
   When this resource is destroyed or updated or refreshed, no actions are done either on CatalystCenter or on devices'
```

Comment:
Can we add a dot at the end of the sentence? :)

Reply:
done